modules/installation-alibaba-config-yaml.adoc

```diff
@@ -37,26 +38,31 @@ metadata:
     hostPrefix: 23
   machineNetwork:
   - cidr: 10.0.0.0/16
-  networkType: OVNKubernetes <2>
+  networkType: OpenShiftSDN
```

Comment:
This is not correct. The default value is `OVNKubernetes`, which was introduced in 4.12 by https://github.com/openshift/openshift-docs/pull/54319

Reply:
I agree completely. This file has been refreshed and my changes re-entered.